### DISCUSSION TITLE
How to use it with vite+vue ?)

### DISCUSSION THREAD

**Discussion by ivanq3w** [JonasKruckenberg/imagetools#214] [Q&A]
Was trying different approaches - does not work :)

**Comment by ivanq3w** [comment]
Hey, @JonasKruckenberg I created a repo https://codesandbox.io/s/vite-vue-imagetools-1nlsx
To reproduce vite+vue+imagetools.
What is a better way to use image sources in <img> and in css ?)

**Reply by ivanq3w** [reply]
And it cannot be build, some errors with vite importing assets :(

**Reply by JonasKruckenberg** [reply]
Sorry for the late reply, and no; `v-bind` in css is pretty new and the way to go about this.
